.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,85 @@
+    runs-on: ubuntu-latest
+    needs: [ unit-tests, integration-tests ]
+    steps:
+      - uses: ./.github/actions/setup-maven
+      - name: Generate Final Coverage Report
+        working-directory: apps/backend
+        run: |
+          mvn verify -DskipTests=true
+      - name: Analyze with Sonarqube
+        uses: SonarSource/sonarqube-scan-action@v5
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)